.github/workflows/ci.yml

```diff
@@ -16,8 +16,30 @@ env:
   RUSTFLAGS: -D warnings -F unused_must_use
 
 jobs:
+  check:
+    name: Clippy (nightly)
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
```

Comment:
I think it would be clearer if this had the matrix removed and replaced with `runs-on: macos-latest`. It only has 1 variant and we have to check for it again on L36 for the `if` guard. Hardcoding it to `runs-on: macos-latest` and dropping the guard seems easier.

Similarly I don't think it's fair to call this "Clippy (nightly)" which might give the impression it's nightly clippy for all targets when it's only testing MacOS + aarch64-apple-tvos. WDYT about naming it "Clippy (tvOS)"?

Reply:
I meanly left it open for when other tier 3 target support is added like visionOS or watchOS targets.
Maybe name it "tier 3"?

@phatblat will likely be along at some point to add visionOS support here after rust core supports it.

Reply:
Yeah I will drop the matrix part.

Reply:
I removed the matrix and updated the naming to have Zbuild-std as that is the real reason that nightly is required for the tier 3 targets.

Reply:
Great, thank you!